platform/security/postgresql.yaml

```diff
@@ -61,11 +61,11 @@ spec:
       accessModes:
       - ReadWriteOnce
       annotations:
-        helm.sh/resource-policy: keep
+        helm.sh/resource-policy: delete
```

Comment:
Don't use this annotation since we are already declaring the reclaimPolicy in storageclass